python_ta/checkers/missing_space_in_doctest_checker.py

```diff
@@ -0,0 +1,61 @@
+"""Checker for E9973 missing-space-in-doctest"""
+import astroid
+
+from pylint.interfaces import IAstroidChecker
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import check_messages
+
+DOCTEST = ">>>"
+
+
+class MissingSpaceInDoctestChecker(BaseChecker):
+    __implements__ = IAstroidChecker
+
+    name = 'missing_space_in_doctest'
+    msgs = {'E9973': ('Space missing in doctest(s) in the docstring of function "%s."',
+                      'missing-space-in-doctest',
+                      'Used when a doctest is missing a space before the code to be executed')
+            }
+    # This is important so that your checker is executed before others
+    priority = -1
+
+    @check_messages("missing-space-in-doctest")
+    def visit_functiondef(self, node: astroid.FunctionDef) -> None:
+        """Visit a function definition"""
+        docstring = node.doc
+
+        if docstring is not None:
+            start_line = node.lineno
+            lines = docstring.split('\n')
+            line_no = start_line
+
+            for line in lines:
```

Comment:
It's a little cleaner to use `enumerate` here, so that you don't need to use a manual `line_no` counter.